WebAPI/Controllers/WebComponentsController.cs

```diff
@@ -65,7 +65,8 @@ public async Task<IActionResult> GetMeeting()
             }
 
             text = text.Replace("#--API_URL--#", apiUrl);
-
+            text = text.Replace("#--YEAR--#", year);
```

Comment:
#--MEETING_YEAR--#, #--MEETING-SEQUENCE_NUM--# could be more descriptive

Reply:
fixed